cognite/powerops/clients/cogshop/data_classes/_processing_logs.py

```diff
@@ -5,47 +5,44 @@
 from cognite.client import data_modeling as dm
 from pydantic import Field
 
-from cognite.powerops.client.dm_client.data_classes._core import (
-    CircularModelApply,
-    DomainModel,
-    InstancesApply,
-    TypeList,
-)
+from ._core import CircularModelApply, DomainModel, InstancesApply, TypeList
 
 __all__ = ["ProcessingLog", "ProcessingLogApply", "ProcessingLogList"]
 
 
 class ProcessingLog(DomainModel):
     space: ClassVar[str] = "cogShop"
+    error_message: Optional[str] = Field(None, alias="errorMessage")
```

Comment:
@doctrino Just curious (definitely out of scope in this PR), why not use pydantic's "alias_generator" feature?

Reply:
Because `pygen` does not assume that the fields are all camel case. The user might mix different styles, and thus an alias is set for each field.